### ISSUE TITLE
REPL social media embed

### ISSUE THREAD

**Issue by hzoo** [babel/website#1383]
https://twitter.com/left_pad/status/909088586473115648

When you post a link to the repl on twitter and elsewhere, we should change what it shows? Maybe it should just not have the same picture/etc (just remove it entirely), it's annoying?

**Comment by xtuc** [comment]
I'm wondering how you could generate a picture of the code (without a backend?).

Either we should remove the cover because it's annoying or an picture with "click to see the REPL" or something.

**Comment by asthas** [comment]
@hzoo I want to do it!

**Comment by hzoo** [comment]
Ok I didn't really go into the solutions - do you understand the issue though? It's just that either we
- remove a picture entirely for the `/repl` URLs
- somehow generate an image (seems unlikely)
- use a different image just for those URLs

I think removing the image for social media seems reasonable (simple, easiest)

**Comment by asthas** [comment]
@hzoo Even I feel the same that removing the pictures for repl is the best possible solution. So should I implement it?

**Comment by hzoo** [comment]
Great, sounds good.

I believe the image is from https://github.com/babel/website/blob/master/_includes/head.html, but not sure how you remove it from the repl link (I guess another question is whether we would want to remove it entirely)

I guess jekyll may have a way to check the `page.url` first

Oh wait I already see a `  {% if page.permalink != '/repl/' %}`

**Comment by asthas** [comment]
@hzoo  I agree, doing it in jeklly by conditionally removing the meta-tag for repl links would be the ideal solution in my mind

**Comment by asthas** [comment]
@hzoo I feel the error is because of lines 31, 33, 37, these would also need a repl check.
